### ISSUE TITLE
Cannot pick media when installed via Flatpak

### ISSUE THREAD

**Issue by charlag** [bleakgrey/tootle#125]
Hey!

tl;dr: I cannot pick files when installed via Flatpak, probably because of the sandbox.

Thanks for the really nice app it's visible that you put so much effort into it and it's rally appreciated.
I also appreciate that it's on Flathub, it just smashed the barrier of trying it out for me.

There's a little bit of a problem with file picking. When I press file picking icon it open not-really-native file picker with empty home folder and nothing else.
When I try to drag'n'drop file into editor it just inserts file URI.

It seems to be a common case for Flatpak app (like KeePassXC had the same problem for me).

Tootle: 0.2.1
Fedora: 29
Flatpak: 1.0.6 (4.fc29)

Let me know if I can help with anything!

**Comment by charlag** [comment]
Hint: `FileChooserNative` may help
https://github.com/Cogitri/gxi/releases/tag/v0.4.5

**Comment by cagatay-y** [comment]
Thanks for the report! The Flatpak package is maintained at [a seperate repo](https://github.com/flathub/com.github.bleakgrey.tootle/). For the issue, there is already the report https://github.com/flathub/com.github.bleakgrey.tootle/issues/1. You can work around the issue until it is resolved by running with `flatpak run --filesystem=home  com.github.bleakgrey.tootle`, which will give the app access to home directory.

**Comment by charlag** [comment]
I would prefer a solution like `FileChooserNative` rather than giving access to the whole filesystem (that's what Flatpak should do in my understanding) but do as you think it better, I have no experience in this field.

Feel free to close it it should be opened in that repo only.
